internal/providers/terraform/provider.go

```diff
@@ -102,3 +102,22 @@ func generatePlanJSON(dir string, path string) ([]byte, error) {
 
 	return out, nil
 }
+
+func CountSkippedResources(resources []*schema.Resource) (map[string]int, int, int, int) {
+	skippedCount := 0
+	unSupportedCount := 0
+	freeCount := 0
+	unSupportedTypeCount := make(map[string]int)
+	for _, r := range resources {
+		if r.IsSkipped {
+			skippedCount++
+			// FIXME: Count free resources when https://github.com/infracost/infracost/issues/121 is done
```

Comment:
@sinabakh / @alikhajeh1 do we think we will add support for this any time soon? If yes then it makes sense to leave this in, but if not, I'd be tempted to remove the logic just now.

Reply:
+1 on removing this FIXME comment line, we don't need the free-resource logic just now.

I see #121 as reviewing/tweaking the existing resources so we support cost components for anything we can easily handle, I guess https://github.com/infracost/infracost/pull/126 is doing that for EC2.

Reply:
I think we will add the free resources soon, but for now, I will remove the whole free logic and will add it back in its related PR.

Reply:
Done